service/src/main/java/uk/gov/mca/beacons/api/export/mappers/ExportMapper.java

```diff
@@ -40,14 +40,17 @@ public ExportMapper(AddressMapper addressMapper) {
   public LabelDTO toLabelDTO(Registration registration) {
     Beacon beacon = registration.getBeacon();
     BeaconUse mainUse = registration.getMainUse();
+    String lastModifiedDate = beacon.getLastModifiedDate() == null
```

Comment:
We shouldn't be defaulting this to a date here, I think if it's null, keep it as null.